File: C/lab7/sort.c
#include <stdio.h>
int bubblesort(int *array, int size);
int main(){
   int size = 5;
   int array[5] = {1,4,5,3,2};
   int i=0;
   printf("=====original=====\n");
   for (i=0; i <size;i++){
      printf("%d\n",array[i]);
   }    
   bubblesort(array, size);
   printf("=====new====\n");
   for (i = 0; i < size; i++){
      printf ("%d\n",array[i]);
   }
   return 0;
}




int bubblesort(int *array, int size){
   int i =0;
   int temp = 0;
   int j;
   for (i =0; i<size; i++){
      for (j=i+1;j<size;j++){
         if (array[i] > array[j]){
            temp = array[i];
            array[i] = array[j];
            array[j] = temp;
         }
      }
   }      

   if (swapped == 0){
      return 0;
   }
   else{
      return 1;
   }
}
      
      
   

File: C/lab7/sort.py
def bubblesort(inlist):
   i =0
   swapped = False 
   temp = 0
   j =0
   for i in range(len(list)-1,0,-1): 
      for j in range(i):
      for (j<len(array)):
         if list[j] > list[j+1]:
            temp =list[j]
            list[j] = list[j+1]
            list[j+1] = temp
            swapped = True
   if swapped == True:
      return [True, array]
   else:
      return [False,array]

size = 5
array[5] = [1,4,5,3,2]
i = 0
print("=====original=====\n")
for i in range(0,size,1):
   print(str(array[i])


bubblesort(array)
print("=====new====\n")
print(str(array))
